app/assets/javascripts/discourse/app/controllers/composer.js

```diff
@@ -764,51 +764,57 @@ export default Controller.extend({
       }
     },
 
-    groupsMentioned(groups) {
+    groupsMentioned({ name, userCount, maxMentions }) {
       if (
-        !this.get("model.creatingPrivateMessage") &&
-        !this.get("model.topic.isPrivateMessage")
+        this.get("model.creatingPrivateMessage") ||
+        this.get("model.topic.isPrivateMessage")
       ) {
-        groups.forEach((group) => {
-          let body;
-          const groupLink = getURL(`/g/${group.name}/members`);
-          const maxMentions = parseInt(group.max_mentions, 10);
-          const userCount = parseInt(group.user_count, 10);
-
-          if (maxMentions < userCount) {
-            body = I18n.t("composer.group_mentioned_limit", {
-              group: `@${group.name}`,
-              count: maxMentions,
-              group_link: groupLink,
-            });
-          } else if (group.user_count > 0) {
-            body = I18n.t("composer.group_mentioned", {
-              group: `@${group.name}`,
-              count: userCount,
-              group_link: groupLink,
-            });
-          }
+        return;
+      }
 
-          if (body) {
-            this.appEvents.trigger("composer-messages:create", {
-              extraClass: "custom-body",
-              templateName: "education",
-              body,
-            });
-          }
+      maxMentions = parseInt(maxMentions, 10);
+      userCount = parseInt(userCount, 10);
+
+      let body;
+      const groupLink = getURL(`/g/${name}/members`);
+
+      if (maxMentions < userCount) {
```

Comment:
I find conditions like "count > max" much easier to read than "max < count"

```suggestion
      if (userCount > maxMentions) {
```